jac-uk/admin
# Feature/2097 outreach report

## What's included?

**Only Sean Cuddihy needs to test this!**

https://jac-apply-admin-production--2097-outreach-j3l1v5bc.web.app [expires 2023-10-20 11:09:48]

**Please remember to press the 'refresh' button for the report!**

We currently ask candidates the following questions when applying:

-  where they heard about the role
- have they attended an outreach event on JAC selection exercises
- did they participate in a Judicial Workshadowing Scheme
- have they taken part in the Pre-Application Judicial Education Programme (PAJE)

The outreach report collates the figures for the answers to these questions.

Closes #2097 

## Who should test?
Sean Cuddihy

## How to test?
Go to the Outreach Report and ensure the report numbers accurately reflect the answers from the applications.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
